website/content/en/status/report-2023-04-2023-06/mfsbsd.adoc

```diff
@@ -20,9 +23,9 @@ This project aims to address that problem.
 
 ==== Location
 
-This project is a GSoC 2023 (Google Summer of Code) project, under mentorship from mailto:otis@FreeBSD.org[Juraj Lutter] and mailto:jrm@FreeBSD.org[Joseph Mingrone].
+This is a GSoC 2023 (Google Summer of Code) project. As such, the official coding period is between May 29, 2023 and August 28, 2023.
+As a humble beginner in the open-source community, the author welcomes all comments / suggestions / pull requests in the project repository], which will be the location for all code throughout this period.
 
-As a GSoC project, the official coding period is between May 29, 2023 and August 28, 2023.
-As a humble beginner in the open-source community, the author of this project welcomes all comments / suggestions / pull requests to the link:https://github.com/soobinrho/freebsd-src/tree/integrate-mfsBSD-building[project repository], which is where all the code for this project will be located for the duration of the official GSoC coding period.
+Mentors: {otis} and {jrm}.
```

Comment:
Sorry, a final question: would you like the mentors to be named _with_, or _without_ visible email addresses? 

I.e. (without): 

> `Mentors: {otis-name} and {jrm-name}`

Reply:
Yes, it makes sense. Thank you for the suggestions 👍👍

Reply:
Resolved by [`0bf8d05` (#207)](https://github.com/freebsd/freebsd-doc/pull/207/commits/0bf8d05a14c309a9f32885446e03bd596deb772b)